backend/src/main/java/com/benine/backend/http/jetty/CameraRequestHandler.java

```diff
@@ -0,0 +1,63 @@
+package com.benine.backend.http.jetty;
+
+import com.benine.backend.ServerController;
+import com.benine.backend.camera.CameraController;
+import com.benine.backend.database.Database;
+import org.eclipse.jetty.server.Request;
+import org.eclipse.jetty.server.handler.AbstractHandler;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * Handles all requests requiring the camera ID.
+ */
+public abstract class CameraRequestHandler extends AbstractHandler {
+
+  /**
+   * Returns cameracontroller
+   * @return cameracontroller interacting with.
+   */
+  protected CameraController getCameraController() {
+    return ServerController.getInstance().getCameraController();
+  }
+  
+  /**
+   * Returns the database
+   * @return database to retrieve information from.
+   */
+  protected Database getDatabase() {
+    return ServerController.getInstance().getDatabase();
+  }
+
+
+  /**
+   * Fetches camera id from http exchange.
+   * @param request the request to fix the id from.
+   * @return the id of the camera.
+   */
+  public int getCameraId(Request request) {
+    // Get path
+    Pattern pattern = Pattern.compile(".*/(\\d*)/.*");
```

Comment:
I think this might go wrong when when an request is made to /camera/1/preset/3/. The pattern .*/camera/(\\d*)/.* or something similar might be better suited.